File: problems/997/solution.py
class Solution:
    def findJudge(self, N: int, trust: List[List[int]]) -> int:
        people = [0] * N
        
        for a, b in trust:
            people[a - 1] -= 1
            people[b - 1] += 1
            
        for i in range(len(people)):
            if people[i] == N - 1:
                return i + 1
        
        return -1

File: problems/997/solution.go
package main
func findJudge(N int, trust [][]int) int {
    people := make([]int, N)
    
    
    for _, ab := range trust {
        a, b := ab[0], ab[1]
        
        people[a - 1]--
        people[b - 1]++
    }
    
    for i := range people {
        if people[i] == N - 1 {
            return i + 1
        }
    }
    
    return -1
}